kiosk/src/Components/ScanQR.tsx

```diff
@@ -70,9 +70,13 @@ const ScanQR: React.FC<IProps> = ({ kiosk }) => {
                 await addGameToKioskAsync(kioskId, shareId);
                 tickEvent("kiosk.submitGameId.submitSuccess");
                 kiosk.navigate(KioskState.QrSuccess);
-            } catch (error) {
-                // it's possible that if it reaches here, the kiosk id was invalid ( someone just puts something random in the url, we should have a toast for this)
-                console.log("Unable to add game to kiosk. Please try again later");
+            } catch (error: any) {
+                setAddingError(error.toString());
+                if (error.toString().includes("404")) {
+                    setErrorDesc("This is likely because the kiosk code no longer exists. Go back to the kiosk to make a new code.");
```

Comment:
I'd suggest making this language less uncertain about what might have been the cause of the 404. Something like "That kiosk code expired. Go back to the kiosk and generate a new one."

We can just say it expired, or wasn't found.